### ISSUE TITLE
Not able to remove extra spacing fusiontime charts

### ISSUE THREAD

**Issue by nikita2423** [fusioncharts/angular-fusioncharts#151]
I am using the fusion time charts for one of my apps, but I am facing an issue regarding extra space around the charts and am not able to find any attribute to control it 

<img width="1444" alt="Screenshot 2022-06-14 at 9 12 01 AM" src="https://user-images.githubusercontent.com/16168752/173488943-dad5a18e-857b-4f07-9357-2109e733999f.png">

**Comment by omanzelli** [comment]
Did you already tried the [Chart Padding & Margins](https://www.fusioncharts.com/dev/chart-attributes/area2d) attributes?

**Comment by nikita2423** [comment]
Yes! but it only applied to **FusionCharts Suite XT**, not in **FusionTime** and i am using FusionTime
